### ISSUE TITLE
[BUG] Builds are broken because integration tests are failing due to dashbaords version number switch

### ISSUE THREAD

**Issue by peternied** [opensearch-project/security-dashboards-plugin#984]
**What is the bug?**
Integration test ci check is failing, see this as an [example](https://github.com/opensearch-project/security-dashboards-plugin/pull/895).  It looks like the Dashboard version number is out of alignment with the expected version in the branch.

```
    Failed to initialize plugins:
    	Plugin "securityDashboards" is only compatible with OpenSearch Dashboards version "2.0.0", but used OpenSearch Dashboards version is "3.0.0". (incompatible-version, /home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/plugins/security-dashboards-plugin/opensearch_dashboards.json)
```

**Comment by kavilla** [comment]
Yes if you migrate to stuff like this: https://github.com/opensearch-project/security-dashboards-plugin/blob/2.0/opensearch_dashboards.json#L3

to just be `2.0.0.0` then it will succeed.

**Comment by peternied** [comment]
@DarshitChanpura any updates on getting this resolved?

**Comment by DarshitChanpura** [comment]
This will be fixed in #989
